datadog_checks_dev/datadog_checks/dev/tooling/commands/meta_commands/prometheus.py

```diff
@@ -0,0 +1,279 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+from collections import defaultdict
+from itertools import product
+
+import click
+import pyperclip
+import requests
+from six import iteritems, itervalues
+from six.moves import input, range
+
+from ..utils import CONTEXT_SETTINGS, abort, echo_info, echo_success, echo_waiting, echo_warning
+from ...constants import get_root
+from ....utils import dir_exists, ensure_unicode, path_join, write_file_lines
+
+METRIC_SEPARATORS = ('.', '_')
+TYPE_MAP = {
+    'gauge': 'gauge',
+    'counter': 'count',
+    'histogram': 'gauge',
+}
+
+
+def sanitize_endpoint(endpoint):
+    if not endpoint.startswith('http'):
```

Comment:
Any reason to exclude the `https` case here?

Reply:
Same thing?

Reply:
Oh... turns out `https` starts with `http`, who knew. Ignore my comment then 🙂